Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1039,6 +1051,14 @@ WarpXOpenPMDPlot::SetConstParticleRecordsEDPIC (
     }
     currSpecies["charge"][scalar].makeConstant( charge );
     currSpecies["mass"][scalar].makeConstant( mass );
+    //   convention: in 1D3V and 2D3V, omitted positions are set to zero
+#if defined(WARPX_DIM_1D_Z)
+    currSpecies["position"]["x"].makeConstant( 0._prt );
+    currSpecies["position"]["y"].makeConstant( 0._prt );
+#endif
+#if defined(WARPX_DIM_XZ)
+    currSpecies["position"]["y"].makeConstant( 0._prt );
```

Comment:
maybe use the variable from the input parameter `<species_name>.y_m` for gaussian beam? (and `<species_name>.x_m` in 1D)

Reply:
I think that would not be ideal, as it breaks the current geometry assumption:
https://github.com/ECP-WarpX/WarpX/pull/4227#issuecomment-1693626344

Reply:
We can add a warning to  the input of `y_m` in 2D3V (and lower d): warn the user that the value is overwritten with 0 if they set it to something else.

Reply:
If we have to follow the convention, could it be possible to not read `y_m, y_rms` in 2D (and `x_m, x_rms` in 1D) from the inputs file and directly impose the values?

Reply:
Or make it optionnal and if one were to parse these parameters so we should implement this warning message.

Reply:
yes, that's what I mean as well. Would be a good follow-up to warn the user if they set it to non-zero and run a 1D/2D sim. Do you want to take that on? :)